Stripe/STPPaymentOptionsViewController.m

```diff
@@ -166,6 +175,19 @@ - (void)updateAppearance {
 }
     
 - (void)finishWithPaymentOption:(id<STPPaymentOption>)paymentOption {
+    BOOL isReusablePaymentMethod = [paymentOption isKindOfClass:[STPPaymentMethod class]] && ((STPPaymentMethod *)paymentOption).type == STPPaymentMethodTypeCard;
```

Comment:
Why not `[paymentOption isKindOfClass:[STPPaymentMethod class]] && [paymentOption isReusable]`?